credentialproviderpackage/pkg/configurator/linux/linux.go

```diff
@@ -22,31 +22,34 @@ import (
 var credProviderTemplate string
 
 const (
-	binPath          = "/eksa-binaries/"
-	basePath         = "/eksa-packages/"
-	credOutFile      = "aws-creds"
-	mountedExtraArgs = "/node-files/kubelet-extra-args"
-	credProviderFile = "credential-provider-config.yaml"
+	binPath               = "/eksa-binaries/"
+	basePath              = "/eksa-packages/"
+	credOutFile           = "aws-creds"
+	mountedExtraArgs      = "/node-files/kubelet-extra-args"
+	ubuntuLegacyExtraArgs = "/node-files/kubelet-extra-args-legacy"
```

Comment:
Just to keep naming consistent
```suggestion
	ubuntuLegacyExtraArgs = "/node-files/ubuntu-legacy-kubelet-extra-args"
```